package.json

```diff
@@ -43,7 +43,6 @@
     "fontsource-ibm-plex-sans": "^3.1.5",
     "i18n": "^0.13.3",
     "is-fqdn": "^2.0.1",
-    "jitsi-iframe-api": "^1.0.0",

```

Comment:
I am not sure where it comes from. I think its similar to the official jitsi package https://www.npmjs.com/package/@jitsi/web-sdk which also has been deprecated already. Since the servers are not running at the moment anways, I don't see a good reason to pour in the extra work to get it up and running.

Reply:
I didn't know that servers also not running 😆